### ISSUE TITLE
Adding gesture for navigation drawer 

### ISSUE THREAD

**Issue by 97balakrishnan** [commons-app/apps-android-commons#1151]
**Summary:** 

swipe gesture to open navigation drawer on create account screen has not been added

**Device and Android version:** 

Stock version of android 6.0
 
 **Commons app version:** 

2.6.7

**Comment by diddypod** [comment]
The Sign Up page seems to be a webView, so it's probably beyond the scope of the app to implement swipe gestures there.
However, should we allow users to navigate through the site without logging in? If yes, we should put that option on the login screen.

**Comment by 97balakrishnan** [comment]
Yeah. I think that the entire create account page can be made into an android activity to ensure uniformity in the app.

**Comment by 97balakrishnan** [comment]
Can i start working on it?

**Comment by misaochan** [comment]
I'm okay with an Android activity being used to handle sign ups instead of the webview. I doubt that it will be an easy task, but anyone who is interested in working on it is certainly welcome to do so. :)

I am on the fence re: the nav drawer being accessible in the signup activity. Currently the app is built around the assumption that the user HAS to be logged in. Allowing non-logged in users to navigate the app would require a refactor of several other components AFAIK, and needs more discussion.

**Comment by 97balakrishnan** [comment]
okay. I will try doing this.
